config/update.go

```diff
@@ -0,0 +1,247 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var appName = "multi_repo_extractor"
+var apiURL = "https://api.github.com/repos/codersrank-org/multi_repo_extractor/releases/latest"
+var currentVersion = version{
+	Major: 1,
+	Minor: 2,
+	Patch: 0,
+}
+
+// CheckUpdates checks github to see if there is a new version and if there is one, downloads it.
+func CheckUpdates() {
+	log.Println("Checking for new versions")
+	release, err := getRelease()
+	if err != nil {
+		log.Printf("Couldn't get latest release from Github, skipping update. Error: %s", err.Error())
+		return
+	}
+	latestVersion, err := getLatestVersion(release)
+	if err != nil {
+		log.Printf("Couldn't find the latest version, skipping update. Error: %s", err.Error())
+		return
+	}
+	if shouldUpdate(currentVersion, latestVersion) {
+		log.Printf("Found new version %+v, updating...", latestVersion)
```

Comment:
This is not the usual way how we show versions: `Found new version {Major:1 Minor:1 Patch:0}, updating...
`. It is supposed to be like this: `Found new version v1.1.0, updating...`